packages/create-tldraw/src/main.ts

```diff
@@ -0,0 +1,232 @@
+import { parse as parseArgs } from '@bomb.sh/args'
+import { intro, outro, select, spinner, text } from '@clack/prompts'
+import { mkdirSync, readFileSync, writeFileSync } from 'node:fs'
+import path, { relative, resolve } from 'node:path'
+import process from 'node:process'
+import { Readable } from 'node:stream'
+import picocolors from 'picocolors'
+import * as tar from 'tar'
+import { groupSelect } from './group-select'
+import { Template, TEMPLATES } from './templates'
+import {
+	emptyDir,
+	formatTargetDir,
+	getInstallCommand,
+	getPackageManager,
+	getRunCommand,
+	uncancel as handleCancel,
+	isDirEmpty,
+	isValidPackageName,
+	nicelog,
+	pathToName,
+} from './utils'
+
+const DEBUG = !!process.env.DEBUG
+
+const allTemplates = [...TEMPLATES.framework, ...TEMPLATES.app]
+
+const HELP = `\
+Usage: create-tldraw [OPTION]... [DIRECTORY]
+
+Create a new project using tldraw.
+With no arguments, start the CLI in interactive mode.
+
+Options:
+  -h, --help                 display this help message
+  -t, --template NAME        use a specific template
+  -o, --overwrite            overwrite the target directory if it exists
```

Comment:
ehh, i don't expect `overwrite` to be an option for these sorts of things. people can make a new folder if they need to and remove the old one. i don't want to give people a loaded gun with this option.

Reply:
i'm just matching `npm create vite` here. happy to remove, but imo this option is pretty unambiguous so i'm not sure it's that much of a footgun